### DISCUSSION TITLE
Welcome to the XMOS Software Development Kit discussions!

### DISCUSSION THREAD

**Discussion by keithm-xmos** [xmos/xcore_iot#89] [General]
We’re using Discussions as a place to connect with XCore users. We hope that you:

Ask questions
Share ideas
Contribute

To get started, introduce yourself below and tell us about the projects you are implementing with the SDK.

**Comment by yprilipukhov** [comment]
Hi,

I'm trying to install the SDK on Windows and I got an issue:
"
...
-- FreeRTOS port does not exist for Windows!  No RTOS targets made.
....
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
XSCOPE_ENDPOINT_LIB
    linked by target "xscope2psf" in directory C:/PProduct/DSPC/Xmos/SDK/xcore_sdk/examples/freertos/tracealyzer/host
    linked by target "xscope_host_endpoint" in directory C:/PProduct/DSPC/Xmos/SDK/xcore_sdk/modules/xscope_fileio/xscope_fileio/host
"

This error is displayed after the command:
cmake  -G "NMake Makefiles" -B build_host

Could you please suggest what could be the problem?

Regards,
Yevhenii

**Reply by netkruzer** [reply]
Did you ever get an answer for this?

**Reply by ZengCheng-cn** [reply]
This issue has been open for a while now. Are there any plans to address it?

**Reply by atomicdog** [reply]
Running from the `XTC Tools 15.3.0 Command Prompt` fixes the NOTFOUND issues by setting the expected variables. I still can't get it to build on Win11 yet, though.
